### PR TITLE
Added ci test for export-bundle functionality

### PR DESCRIPTION
**Pull request by vinu2003**
## Description of change
This PR addresses code changes to remove developer flag and added ci test for export bundle command.

### CODE REVIEW

File: acceptancetests/assess_bundle_export.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    new_client.wait_for_started()
+    old_model = get_current_model(client)
+    if old_model != TEST_MODEL:
+        raise JujuAssertionError('Models mismatch : Bundle error.')
+
+    os.remove(model+".yaml")
+
+
+
+def add_model(client):
```
Comment (wallyworld) [inline acceptancetests/assess_bundle_export.py]: These model methods are duplicated code - if used in more than one place they should be put into a helper models.py

File: acceptancetests/assess_bundle_export.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    log.info("Deploying {}".format("mediawiki-simple bundle..."))
+
+    if not os.path.exists(model+".yaml"):
+        raise JujuAssertionError('export bundle command failed to create bundle file.')
+
+    new_client = add_model(client)
+    new_client.deploy(model+".yaml")
+    new_client.wait_for_started()
+    old_model = get_current_model(client)
+    if old_model != TEST_MODEL:
```
Comment (wallyworld) [inline acceptancetests/assess_bundle_export.py]: There needs to be a more comprehensive check than just the model name - perhaps re-export the bundle and compare the contents to the first export.

File: acceptancetests/utility.py [hunk 3]
```diff
@@ -420,3 +423,46 @@ def assert_dict_is_subset(sub_dict, super_dict):
         raise JujuAssertionError(
             'Found: {} \nExpected: {}'.format(super_dict, sub_dict))
     return True
+
+def add_model(client):
```
Comment (wallyworld) [inline acceptancetests/utility.py]: You also need to change existing usages of add_model and current_model to use these new helpers. I think the current usages are in assess_destroy_model.py, but do a check

**Comment by vinu2003** [comment]
$$merge$$

**Comment by anastasiamac** [comment]
[adjust maas provider to use callback to invalidate cloud credential when needed](https://trello.com/c/qvzD0WWc/351-adjust-maas-provider-to-use-callback-to-invalidate-cloud-credential-when-needed)

**Comment by anastasiamac** [comment]
This is not right :) Feel free to ignore :)

**Pull request merged**